src/vulkan-renderer/exception.cpp

```diff
@@ -4,8 +4,11 @@
 
 namespace inexor::vulkan_renderer {
 
-VulkanException::VulkanException(const std::string &message, const VkResult result)
-    : InexorException(message + " (" + vk_tools::result_to_string(result) + ": " +
-                      vk_tools::result_to_description(result) + ")") {}
+VulkanException::VulkanException(std::string message, const VkResult result)
+    : InexorException(message.append(std::string_view(" ("))
+                          .append(vk_tools::as_string<VkResult>(result))
+                          .append(std::string_view(": "))
+                          .append(vk_tools::result_to_description(result))
+                          .append(std::string_view(")"))) {}
```

Comment:
appending string_view seems odd?
why not just `append(" (")` e.g.

Reply:
Done.